vardbg/output/video_writer/config.py

```diff
@@ -106,16 +118,16 @@ def __init__(self, config_path):
         self.font_heading = (sub_path(fonts["heading"]), fonts["heading_size"])
         self.font_intro = (sub_path(fonts["intro"]), fonts["intro_size"])
 
-        style = MonokaiStyle
+        style = DefaultStyle
         self.styles = load_style(style)
 
         self.bg = parse_hex_color(style.background_color)
         self.highlight = parse_hex_color(style.highlight_color)
         self.fg_divider = self.styles[Token.Generic.Subheading]["color"]
```

Comment:
The Base16 colors should be used directly rather than translating them into a Pygments style and referencing that.